### PR TITLE
Handle 'cut' events

### PR DESCRIPTION
**Pull request by evgenykochetkov**

### CODE REVIEW

**Comment by nadbm** [comment]
thank you @evgenykochetkov , mind adding this to the README as well? thanks

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Demo here: https://nadbm.github.io/react-datasheet/
 Examples are located in https://github.com/nadbm/react-datasheet/tree/master/docs/src/examples
 
 Current features:
-* Select cells, copy-paste cells
+* Select cells, cut, copy and paste cells
```
Comment (evgenykochetkov) [inline README.md]: Mentioned "cut" in "Current features", not sure what else needs to be added, since this is a very small feature and no API was changed

Comment (evgenykochetkov) [reply README.md]: @nadbm, should I anything else?

Comment (nadbm) [reply README.md]: @evgenykochetkov no that looks good. I will merge it soon

**Pull request merged**